src/library.py

```diff
@@ -495,3 +495,202 @@ def check_detail(self) -> Dict:
         print("Verification results dict: ")
         print(output)
         return output
+
+
+class AutomaticOADamperControl(CheckLibBase):
+    points = ["o", "m_oa", "eco_onoff"]
+
+    def verify(self):
+        tol = 0.03
+        self.result = ~(
+            (self.df["o"] < tol) & (self.df["m_oa"] > 0) & (self.df["eco_onoff"] == 0)
+        )
+
+    def check_bool(self) -> bool:
+        if len(self.result[self.result == True] > 0):
+            return True
+        else:
+            return False
+
+    def check_detail(self):
+        print("Verification results dict: ")
+        output = {
+            "Sample #": len(self.result),
+            "Pass #": len(self.result[self.result == True]),
+            "Fail #": len(self.result[self.result == False]),
+            "Verification Passed?": self.check_bool(),
+        }
+        print(output)
+        return output
+
+
+class FanStaticPressureResetControl(CheckLibBase):
+    points = ["p_set", "d_VAV_1", "d_VAV_2", "d_VAV_3", "d_VAV_4", "d_VAV_5"]
```

Comment:
I will leave it as is for now and let me figure this out in a new branch. Does this sound ok to you Jeremy?

Reply:
Sounds good!

Reply:
What made you want to split this into multiple verification items?

Reply:
the fact that we need to assume how many vavs are in the model makes this less practical for automated deployment. So if we can set the verification item to only test against a specific VAV and instantiate multiple verification cases, each regarding one VAV, that would be easier to deploy

Reply:
How about we leave it as is for now and try to figure out how to deal with no. of VAVs in another branch? Would it be fine with you? Rather than splitting this verification item, if we can find a way to handle the no of VAVs flexibly, that'd be more practical. However, I'd need a bit more time for this so, I was going to find this solution later considering our deadline.